cmd/release/main.go

```diff
@@ -499,12 +537,12 @@ func calculateNextRelease(now time.Time, latestRelease *github.RepositoryRelease
 	return releaseName, nil
 }
 
-func pluginDownloadURL(plugin *plugin.Plugin, releaseName string) (string, error) {
+func (c *command) pluginDownloadURL(plugin *plugin.Plugin, releaseName string) (string, error) {
 	zipName, err := pluginZipName(plugin)
 	if err != nil {
 		return "", err
 	}
-	return fmt.Sprintf("https://github.com/%s/%s/releases/download/%s/%s", githubReleaseOwner, githubRepo, releaseName, zipName), nil
+	return fmt.Sprintf("https://github.com/%s/%s/releases/download/%s/%s", c.githubReleaseOwner, release.GithubRepoPlugins, releaseName, zipName), nil
```

Comment:
Curious why not use the GH API client?

Reply:
In this case, we're crafting the download URL prior to uploading it via the release API (since it will be part of the `package-releases.json` file). If we wanted to do this after uploading the assets, we'd need to generate the JSON file last and sign it.